lib/Db/ItemMapperV2.php

```diff
@@ -31,15 +32,50 @@ class ItemMapperV2 extends NewsMapperV2
 {
     const TABLE_NAME = 'news_items';
 
+    /**
+     * @var IUserManager
+     */
+    private $userManager;
+
```

Comment:
That makes sense, I agree.. I've picked this method because it allowed me to modify the least amount of code.

There's another option that I've thought of, and I've now committed it. I moved the display name mapping into a function in the ShareService. I call this function from the ItemController when returning fetched items.

Is that better?

Reply:
Sounds good

Reply:
Great, everything is pushed! Resolving this comment